scripts/data/exchange/soap-template.ts

```diff
@@ -1,14 +1,26 @@
-const getMailboxData = (emails) => {
-  return emails.map(email =>
-    `<t:MailboxData>
+function getMailboxData(emails: string[]) {
```

Comment:
Is this code tested? I'm afraid of changing code in the exchange logic since we're not using it atm.